testcafe/testcafe.config.ts

```diff
@@ -1,3 +1,4 @@
 export class Config {
   static baseUrl = "http://localhost:8080";
+  static baseTestCovmapUrl = "http://test.covmap.de";
```

Comment:
what is missing to do this locally?

Reply:
It's a mistake, I've foggot to fix the "test.ts" file. Should work locally.

Reply:
fix has been commited.

Reply:
The second URL still seems to be in the config and unused.

Reply:
second URL also removed.